src/main/java/at/hannibal2/skyhanni/config/ConfigManager.kt

```diff
@@ -203,6 +203,11 @@ class ConfigManager {
         if (output == defaultValue) {
             logger.log("Setting $fileName to be blank as it did not exist. It will be saved once something is written to it")
         }
+        @Suppress("SENSELESS_COMPARISON")
+        if (output == null) {
```

Comment:
defaultValue cant be null, output should never be null here as well. what edge case is this?
if we call this function with defaultValue null, it should throw an error elsewhere before touching this code, right?

Maybe we should add `var output: Any? = defaultValue` above?